docs/src/pages/Homepage.vue

```diff
@@ -53,6 +73,83 @@
 import { defineComponent } from 'vue'
 import { useMeta } from 'quasar'
 import { useDocStore } from 'assets/doc-store.js'
+import {
+  mdiAnimation,
+  mdiApplicationExport,
+  mdiCalendar,
+  mdiCardMultiple,
+  mdiFormDropdown,
+  mdiFormTextbox,
+  mdiImageSizeSelectSmall,
+  mdiTable
+} from '@quasar/extras/mdi-v6'
+
+const importantPages = {
+  mostUsedPages: [
+    {
+      label: 'QTable',
+      icon: mdiTable,
+      path: 'vue-components/table'
+    },
+    {
+      label: 'QInput',
+      icon: mdiFormTextbox,
+      path: 'vue-components/input'
+    },
+    {
+      label: 'QSelect',
+      icon: mdiFormDropdown,
+      path: 'vue-components/select'
+    },
+    {
+      label: 'QBtn',
+      icon: 'view_quilt',
+      path: 'vue-components/button'
+    },
+    {
+      label: 'QCard',
+      icon: mdiCardMultiple,
+      path: 'vue-components/card'
+    },
+    {
+      label: 'Flavour',
+      img: 'https://cdn.quasar.dev/logo-v2/svg/logo-mono-cyan.svg',
+      path: 'start/pick-quasar-flavour'
+    }
+  ],
+  pagesToDiscover: [
+    {
+      label: 'quasar.conf.js',
```

Comment:
Use `quasar.config` instead, into the new CLI the file will change name to be this one and will possibly support TS